packages/blockchain-wallet-v4/src/coinSelection/index.js

```diff
@@ -30,8 +31,8 @@ export const isFromAccount = selection =>
 export const isFromLegacy = selection =>
   selection.inputs[0] ? selection.inputs[0].isFromLegacy() : false
 
-export const dustThreshold = feeRate =>
-  Math.ceil((Coin.inputBytes({}) + Coin.outputBytes({})) * feeRate)
+export const dustThreshold = (feeRate, change) =>
+  Math.ceil((Coin.inputBytes(change) + Coin.outputBytes(change)) * feeRate)
```

Comment:
> Coin.inputBytes(change)

I think this is wrong, because the input to create the change may or may not be the same `type` as the change.

Reply:
If you go below this threshold, you consider the change dust and burn it instead of adding an extra output for it.
I'm not sure why it's considering an extra input here either